my-app/src/app/shared/pipes/duration.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'duration'
```

Comment:
consider to add `pure: true`, see https://angular.io/api/core/Pipe

Reply:
The documentation says "Pipes are pure by default."

Reply:
sorry, my bad, they finally applied the default to `true`